pages/api/openPR.js

```diff
@@ -191,13 +204,13 @@ function submitPullRequests(projectName, filesObject) {
 export default async (req, res) => {
   if (req.method === "POST") {
     let values = req.body.values;
-    let nomineeJSON, sdgNumber, evidenceText, sortedSubmission;
+    let nomineeJSON, sortedSubmission;
 
     // Exclude contact information from pull request
     delete values["contact"];
 
     // Parse SDG information
-    values = getSDGRelevanceInfo(values, sdgNumber, evidenceText);
```

Comment:
@nathanbaleeta: you were essentially passing empty values. instead of initializing these variables in the main function and passing them empty, these are no longer passed and initialized only where they are used